src/skills-builder/skills-builder-modal/view-results/CarouselStack.jsx

```diff
@@ -35,6 +36,19 @@ const CarouselStack = ({ selectedRecommendations }) => {
     </h3>
   );
 
+  const handleCourseCardClick = (courseKey, productType) => {
+    sendTrackEvent(
+      'edx.skills_builder.recommendation.click',
+      {
+        category: 'skills_builder',
+        page: 'skills_builder',
```

Comment:
What is the difference between `category` and `page`? How were these values determined? Should they be the same thing? Should page be something more descriptive like `results`?

Reply:
The `page` value comes from the `edx.bi.user.recommended.course.click` event that's being fired in `learner_home`. We wanted to match that one as closely as possible. `category` is used in all of the other events in Skills Builder. That's how they ended up being the same thing. 

In `learner_home`, the `page` is set to `dashboard` so I can see what you mean by setting it to `results` here 🤔